src/vm/eetwain.cpp

```diff
@@ -5913,6 +5913,14 @@ ULONG32 EECodeManager::GetStackParameterSize(EECodeInfo * pCodeInfo)
     } CONTRACTL_END;
 
 #if defined(_TARGET_X86_)
+#if defined(WIN64EXCEPTIONS)
+    if (pCodeInfo->IsFunclet())
```

Comment:
I don't understand why funclets should be handled differently. Since the funclet has no stack argument, it seems that the function should return 0 even without this modification. So it seems that we may be fixing a symptom, but not the root of the issue.